tests/golden/copy-command/copy-command.bats

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env bats
+
+load '../helpers/bats-support/load'
+load '../helpers/bats-assert/load'
+load '../helpers'
+
+@test "copy an entry" {
+  run coffer create /a
+  assert_output "[SUCCESS] Entry created at '/a'."
+
+  run coffer copy /a /b
+  assert_output "[SUCCESS] Copied '/a' to '/b'."
+
+  run cleanOutput coffer view /
+  cat copy-an-entry.gold | assert_output -
+}
+
+@test "copy a directory" {
+  run coffer create /a/b/c
+  assert_output "[SUCCESS] Entry created at '/a/b/c'."
+
+  run coffer copy /a /
+  assert_output "[SUCCESS] Copied '/a/b/c' to '/b/c'."
+
+  run cleanOutput coffer view /
+  cat copy-a-directory.gold | assert_output -
```

Comment:
Is it okay to write like this?
````bash
...
  EXPECTED=$(cat <<EOF
/
  a - [2000-01-01 01:01:01]
  b - [2000-01-01 01:01:01]
EOF
)
  assert_output "$EXPECTED"
...
````
Because I don't know how to write heredocs in `assert_output` command.

Reply:
You can add a `-` to tell `assert_output` to read from stdin, so I think something like this should work:

```
  assert_output - <<EOF
/
  a - [2000-01-01 01:01:01]
  b - [2000-01-01 01:01:01]
EOF
```

There's an example [in the docs](https://github.com/bats-core/bats-assert#standard-input-heredocs-and-herestrings), (although the example is using herestrings instead of heredocs).

Reply:
Done